core/math/math.odin

```diff
@@ -1790,7 +2071,91 @@ atan2 :: proc{
 	atan2_f16le, atan2_f16be,
 }
 
-// Return the arc tangent of x, in radians. Defined on the domain of [-∞, ∞] with a range of [-π/2, π/2]
+/*
+Return inverse of tangent (arctan) of given input.
+
+**Only accept floats**
+
+Relation with math.tan:
+
+    x = tan(y)
+    y = atan(x)
+
+    For real result, x could be any real numbers,
+    resulting single value that lies in following range(s)
+
+        [-π/2, π/2] (in radians)
+        [ -90,  90] (in degrees)
+
+Note:
+    Implementation of math.atan using math.atan2(y,x) where y is the input value,
+    and x is set to 1. See math.atan2 for detail.
+
+Beware of special cases i.e. `-0.0`, `+0.0`, `-Inf`, `+Inf`, and `Nan` (see examples).
+For special cases related math.atan2(x, 1)
+
+    atan2(  0, 1) = 0
+    atan2( -0, 1) = - 0
+    atan2(  ∞, 1) = π/2
+    atan2( +∞, 1) = + π/2
+    atan2( -∞, 1) = - π/2
+    atan2(NaN, 1) = NaN
+
+Inputs:
+- x: input value of type floats in radians
+
+Output:
+- x: ouput value that with same type of the input in radians
+
+Example:
+        x30_f32:     f32 = 30.0;                     tan_x30_f32     := math.tan(x30_f32);     atan_x30_f32     := math.atan(tan_x30_f32)     // without converting to radians
```

Comment:
This is going to look horrible in the text editor with wrapping on documentation snippets, and would require some scrolling on the website. Maybe you should wrap these manually